src/views/components/display-html/DisplayHTML.native.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Html from 'react-native-render-html';
+import { string } from 'prop-types';
+import { Box } from '../../components';
+
+class DisplayHTML extends Component {
```

Comment:
I think we should refactor the name to just be `Html` (just the H capitalised too). That way, there's no confusion on *how* the HTML will be displayed, just that this component will simply render out any HTML passed to it.